cmd/influxd/server_integration_test.go

```diff
@@ -600,6 +600,7 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		if name == "" {
 			name = tt.query
 		}
+		fmt.Printf("TEST: %d: %s\n", i, name)
```

Comment:
You don't, the problem is that `test -v` Doesn't put the test info inline with the actual output of the server. Basically, you get all the server output, and then at the end you get the test output. So it's impossible to see which maps with what. With this you see the info about the test running with the server output. Besides, this only shows up if you run with `-v`